pkg/config/load_test.go

```diff
@@ -184,27 +185,78 @@ mirror:
   - catalog: registry.com/ns/baz:v1.2
     targetTag: v1.3
   - catalog: registry.com/ns/baz:v1.2
+  - catalog: oci:///tmp
```

Comment:
Some unit tests for GetUniqueName are in pkg/api/v1alpha2/types_config.go
Do you think it makes sense to move these tests there?
TBH I don't understand what load_test.go is for.

Reply:
This test loads the config once and then tests each entry with GetUniqueName. So it tests the loading functionality in a round about way. The other test in pkg/api/v1alpha2/types_config.go just uses static config objects. Leaving here for now